app/controllers/confirmations_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ConfirmationsController < ApplicationController
+
+def new
+  @user = current_user
```

Comment:
no need to set `current_user` to an instance variable here. your `current_user` helper method will be available in all views 👍